### DISCUSSION TITLE
How to prevent flash.nvim's search from ruining my buffer

### DISCUSSION THREAD

**Discussion by txtyash** [LazyVim/LazyVim#1148] [Q&A]
When I run a fast search using `/` flash.nvim exits search if there are no matches and starts the flash search. I end up pressing keys after it has automatically exited search. This is very annoying. How do I stop this from happening?

**Comment by yusufaktepe** [comment]
See https://github.com/LazyVim/LazyVim/issues/1109#issuecomment-1633740572
